parser/parser.y

```diff
@@ -14730,6 +14731,23 @@ PlanReplayerStmt:
 			Limit:      nil,
 		}
 
+		$$ = x
+	}
+
+/********************************************************************
+ *
+ * Plan Chnage Capture Statement
+ *
+ * PLAN REPLAYER Change 'begin_time' 'end_time'
```

Comment:
Can we use `now()` as the `end_time` here?

Reply:
And `plan change capture now()-3d, now()` is ok?

Reply:
I think we can make `plan change capture colloect <ts1>` as `plan change capture collect <ts1> now()`, but I think support `now()-3d` is not necessary.